yarn-project/end-to-end/src/e2e_get_notes.test.ts

```diff
@@ -0,0 +1,86 @@
+import { AztecAddress, Wallet, toBigInt } from '@aztec/aztec.js';
+import { TestContract } from '@aztec/noir-contracts';
+
+import { setup } from './fixtures/utils.js';
+
+describe('e2e_get_notes', () => {
+  // export DEBUG=aztec:e2e_get_notes
```

Comment:
nit: not needed?

Reply:
Ah thanks, I think I just copied this from another file when setting up the boilerplate and then forgot to figure out what it was :stuck_out_tongue: